tracer/test/snapshots/ManualInstrumentationTests.ManualAndAutomatic.verified.txt

```diff
@@ -2,6 +2,24 @@
   {
     TraceId: Id_1,
     SpanId: Id_2,
+    Name: initial,
```

Comment:
This is the only _actual_ change in the snapshot, the rest are just bumping of IDs